### ISSUE TITLE
Cancel an open game you created

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#1733]
[This may already be an issue -- I couldn't find it listed]

Sometimes I create an open game by mistake when I mean to invite a particular person. I'd like the ability on the Open Games page to cancel a game I've already created.

**Comment by cgolubi1** [comment]
On Thu, 08 Oct, 2015 at 08:33:33 -0700, dwvanstone wrote:

> [This may already be an issue -- I couldn't find it listed]
> 
> Sometimes I create an open game by mistake when I mean to invite a particular person. I'd like the ability on the Open Games page to cancel a game I've already created.

Your solution is probably a good idea, but it also made me think ---
i accidentally create open games all the time while testing.  If we
think the more common case is to challenge a specific opponent, i
wonder if making the UI harder to use would make it easier to use right.
In particular, what if "Any player" wasn't an element in the drop-down
list, but instead there was a box next to the drop-down list that you
had to check to create an open game, and the default player was blank.
Then if you didn't touch the opponent field at all, the UI would give
you an error and send you back to specify an opponent.

**Comment by dwvanstone** [comment]
This would also be a nice consistent with the Rejected/Canceled issue -- I'd like to see it part of the mix for December.

**Comment by blackshadowshade** [comment]
I've been thinking about this. To fix the initial problem that @dwvanstone filed, would it be better to allow a game creator to assign an opponent to an open game?

**Comment by blackshadowshade** [comment]
As a follow-up, if we implement my idea, would we then need to be able to cancel open games?

(There would be a kludgy workaround if someone really wanted to do so: add an opponent, then withdraw the game.)

**Comment by dwvanstone** [comment]
I think of a few reasons why I'd want to cancel an open game:

- I created an open game, but I wanted to make it out of 5 instead of out of 3. 
- I chose the wrong buttons
- I created a game that nobody picked for two weeks and I it's embarrassing me that it's still there

Now none of these cases have come up for me in the past year -- only the case where I meant to assign a person. But it does seem like a kludgy workaround to add an opponent then withdraw.

However, this does seem to be a pretty good solution to the initial problem.
